documentation/asciidoc/topics/ref_upgrades.adoc

```diff
@@ -0,0 +1,18 @@
+[id='upgrades_{context}']
+= {brandname} cluster upgrades
+
+[role="_abstract"]
+{ispn_operator} can automatically upgrade {brandname} clusters when new versions become available.
+You can also perform upgrades manually if you prefer to control when they occur.
+
+[NOTE]
+====
+{ispn_operator} requires the Operator Lifecycle Manager to perform cluster upgrades.
+====
+
+[discrete]
+== Upgrade notes
+
+If you upgrade {brandname} clusters manually, you should upgrade to the latest {brandname} 12.x version if you have already upgraded {ispn_operator} from 2.0.x to 2.1.x, which corresponds to {brandname} Server 11.x to 12.x.
```

Comment:
Maybe we should reword this to:

"If you upgrade {brandname} clusters manually and have upgraded {ispn_operator} channel from 2.0.x to 2.1.x you should approve the latest {brandname} 12.x version as soon as possible. This is in order to mitigate the impact of ISPN-13116 and prevent data loss from occurring."

Reply:
thanks @ryanemerson I've updated this but slightly reworded